src/lib/components/dragdrop/DragDrop.svelte

```diff
@@ -173,10 +173,10 @@
   {/if}
 
   {#if showFileTypeError}
-    <div class="d-flex gap-2">
+    <div class="d-flex gap-2" style="justify-content:center">
```

Comment:
jc-center class can be used please check for bootstrap class

Reply:
fixed